### PR TITLE
Update to version 0.3.0

### PR DESCRIPTION
**Pull request by kylacochrane**
Update `gasclustering` to version 0.3.0.

See `CHANGELOG.md` for updates.

### CODE REVIEW

**Review by emarinier — APPROVED** [review]

File: assets/samplesheet.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
 sample,mlst_alleles
-sample1,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample1.mlst.json
-sample2,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample2.mlst.json
-sample3,https://raw.githubusercontent.com/phac-nml/gasclustering/dev/tests/data/reports/sample3.mlst.json
+sample1,https://raw.githubusercontent.com/phac-nml/gasclustering/update/input_assure/tests/data/reports/sample1.mlst.json
```
Comment (apetkau) [inline assets/samplesheet.csv:2]: Change test data back to `dev`

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks good. One small change 😄

**Review by apetkau — APPROVED** [review]
Thanks so much.

**Review by apetkau — APPROVED** [review]
Looks great to me. Thanks so much Kyla 😄

**Pull request merged**